.github/workflows/new-release.yml

```diff
@@ -0,0 +1,66 @@
+name: New Release
+
+on:
+  workflow_dispatch:
+    inputs:
+      version:
+        description: 'Version'
+        type: string
+        required: true
+
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+    timeout-minutes: 15
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+      - uses: actions/setup-node@v3
+        with:
+          node-version: 18
+          cache: 'yarn'
+      - name: Get version # TODO(eeeqeee) add option to increment from current version
+        id: get-version
+        run: |
+          echo "version=${{ inputs.version }}" >> $GITHUB_OUTPUT
+      - name: Set git config
+        run: |
+          git config user.name "github-actions[bot]"
+          git config user.email "41898282+github-actions[bot]@users.noreply.github.com"
+      - name: Change version
+        run: |
+          TEMP_FILE=$(mktemp)
+          jq --arg v "${{ steps.get-version.outputs.version }}" '.version = $v' package.json > $TEMP_FILE
+          cp $TEMP_FILE package.json
+          git add .
+          git commit -m "${{ steps.get-version.outputs.version }}"
+      - name: Yarn install
+        run: |
+          yarn install --immutable --immutable-cache
+      - name: Setup .env
+        run: |
+          echo $ENVFILE > apps/frontend/.env.local
+        env:
+          ENVFILE: ${{ secrets.ENVFILE }}
```

Comment:
i think the repo owner is the only one thats able to setup secrets, so we should wait to merge

Reply:
Internet says we can manage secrets via rest API, but yea we can just wait for frzyc too, no rush